wpilibj/src/test/java/edu/wpi/first/wpilibj/controller/ControlAffinePlantInversionFeedforwardTest.java

```diff
@@ -30,15 +28,15 @@ void testCalculate() {
                     0.02);
 
     assertEquals(48.0, feedforward.calculate(
-         new MatBuilder<>(Nat.N2(), Nat.N1()).fill(2, 2),
-         new MatBuilder<>(Nat.N2(), Nat.N1()).fill(3, 3)).get(0, 0),
+         Matrix.mat(Nat.N2(), Nat.N1()).fill(2, 2),
+         Matrix.mat(Nat.N2(), Nat.N1()).fill(3, 3)).get(0, 0),
```

Comment:
There are *tons* of places throughout this PR where `Matrix.mat(..., Nat.N1()).fill(...)` is used instead of the more concise `VecBuilder.fill(...)`. Unless you can make an argument for why we should remove `VecBuilder` and use `Matrix.mat` instead, then I think `VecBuilder.fill` should be used exclusively in these cases.

Reply:
👍 I was just going through and fixing that. VecBuilder is a lot more simple and I dom't see a reason to remove it.